### PR TITLE
MOTECH-2209 Documenting the CSD module and ...

### PR DESCRIPTION
**Pull request by craigappl**
added the following updates to the module documentation
- Updated the copyright date to 2016.
- Rearranged the index to alphabetical order (IVR module needed to be moved up)

### CODE REVIEW

**Comment by craigappl** [comment]
FYI @llavoie Can you code review this?

File: docs/source/csd.rst [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+The CSD Standard and OpenHIE integration
+----------------------------------------
+The CSD is used by `OpenInfoMan <https://github.com/openhie/openinfoman>`_, which is the reference software for the `OpenHIE Health Worker Registry <https://ohie.org/health-worker-registry/>`_. This registry provides a big picture view of a country's health workforce as well as a single source for supporting patient/provider flows in a Health Information Exchange. More information on the CSD standard can be found in the `IHE Care Services Discovery Profile <http://wiki.ihe.net/index.php/Care_Services_Discovery>`_ and the OpenInfoMan implementation of that profile on their `GitHub page <https://github.com/openhie/openinfoman>`_.
+
+In some implementations, OpenInfoMan may be deployed within the OpenHIE stack behind a firewall. `OpenHIM <http://openhim.org/>`_ is the Interoperability layer for OpenHIE. It responds to CSD requests from point of service applications like MOTECH and is able to forward those requests to OpenInfoMan. The same requests to OpenInfoMan can be made to OpenHIM. OpenHIM being the system that provides access controls and auditing of requests.
+
```
Comment (llavoie) [inline docs/source/csd.rst:17]: This last sentence "OpenHIM being the system that provides access controls and auditing of requests" is a fragment. You might just want to rephrase as "OpenHIM is the system that provides access controls and auditing of requests".

**Comment by pgesek** [comment]
test this please

**Comment by pgesek** [comment]
test this please

**Comment by llavoie** [comment]
@craigappl Apologies for letting this one languish. Looks like it needs a merge with latest master.

**Pull request closed without merging**